index.html

```diff
@@ -1,176 +1,780 @@
+  <div id="about" class="about-us section">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-6 align-self-center">
+          <div class="section-heading">
+            <h4>About us</h4>
+            <img src="assets/images/heading-line-dec.png" alt="">
+            <p>At Quanteco we are working towards a future where we can track our entire carbon footprint online, and live our lives mindful of the effects of our lifestyle. We believe that with an understanding of our true impact as individuals, we can harness our competitive and social potential to solve the climate crisis.
+            </br>
+              Using the science of life cycle analysis, we strip away the layers of industry between farm and plate, so that normal people can make decisions that improve the agricultural techniques of the world.</p>
+      
+            </p>
+          <div class="row">
+            <div class="col-lg-6">
+              <div class="box-item">
+                <h5> Robert - CEO </h5>
+               <p><img class="teamimage" src="assets/images/qe/robert.png" style="height: 267px; width: 175px;" alt="Robert"> <br></p>
+              <p>
+               Mathematical engineer <br>
+                Likes: Data<br>
+                Dislikes: Balance fallacy
+                </p>
 
-                // Show the current tab, and add an "active" class to the button that opened the tab
-                document.getElementById(name).style.display = "block";
-                evt.currentTarget.className += " active";
-            }
 
-            // Get the element with id="defaultOpen" and click on it
-            document.getElementById("defaultOpen").click();
+              </div>
+            </div>
+            <div class="col-lg-6">
+              <div class="box-item">
+                  <h5> Evert - CTO </h5>
+                  <p> <img  src="assets/images/qe/evert.png" alt="Evert" style="height: 267px; width: 158px;"> <br></p> 
+                <p>
+                   Computer scientist <br>
+                Likes: Backends <br>
+                Dislikes: Bikeshedding
+                </p>
+              </div>
+            </div>
+           
+            <!--
+            <div class="col-lg-6">
+              <div class="box-item">
+                <h4><a href="#">Fixing Issues About</a></h4>
+                <p>Lorem Ipsum Text</p>
+              </div>
+            </div>
+            <div class="col-lg-6">
+              <div class="box-item">
+                <h4><a href="#">Co. Development</a></h4>
+                <p>Lorem Ipsum Text</p>
+              </div>
+            </div>
+            -->
+             <p> If our goal appeals to you and you would like to advance our cause, please consider <a rel="nofollow" href="https://paypal.me" target="_blank">donating to us via PayPal</a>. Donations over €50 will provide access to the Quanteco discord server where we discuss app development and upcoming features. Thank you.
```

Comment:
Uuuhm we should discuss this right (not that I strongly disagree). Also the paypal link is empty, is this really publish-worthy?

Reply:
Agree. Definitely not publish worthy, just a draft